docs/_docs/themes.md

```diff
@@ -218,13 +218,15 @@ _sass
 
 Your theme's styles can be included in the user's stylesheet using the `@import` directive.
 
+{% raw %}
 ```css
-{% raw %}@import "{{ site.theme }}";{% endraw %}
+@import "{{ site.theme }}";
 ```
+{% endraw %}
 
 ### Theme-gem dependencies
 
-From `v3.5`, Jekyll will automatically require all whitelisted `runtime_dependencies` of your theme-gem even if they're not explicitly included under the `gems` array in the site's config file. (NOTE: whitelisting is only required when building or serving with the `--safe` option.)
+From `v3.5`, Jekyll will automatically require all whitelisted `runtime_dependencies` of your theme-gem even if they're not explicitly included under the `gems` array in the site's config file. (Note: whitelisting is only required when building or serving with the `--safe` option.)
```

Comment:
`gems` array and/or `plugins` arrays..

Reply:
Thanks for noticing, as we mention v3.5, let's just use `plugins` here as we did in #6082